src/main.zig

```diff
@@ -470,6 +476,7 @@ fn lint(
                         "line is {s}{} characters long{s}; the maximum is {}",
                         .{ red_text, len, end_text_fmt, analyzer.max_line_length },
                     ),
+                    .DupeImport => |name| try stdout_writer.print("found duplicate import of {s}", .{name}),
```

Comment:
We can add color highlighting here. 
```suggestion
                    .DupeImport => |name| try stdout_writer.print(
                        "found duplicate import of {s}{s}{s}",
                        .{bold_magenta, name, end_text_fmt},
                    ),
```